recipes/awsebcli/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "awsebcli" %}
+{% set version = "3.18.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a44fb91b3f6b35ea1166200df1954f8dde2228e0e1b37267b2ca6b8fa66809b5
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - botocore >=1.15
```

it seems this package also need ``blessed``
```suggestion
    - botocore >=1.15
    - blessed >=1.9.5
```

but there is no ``blessed`` on conda-forge, you will need to add it as well

if you want to, next time you can give a go to [grayskull :skull: ](https://github.com/marcelotrevisani/grayskull) to generate recipes for packages on pypi

---

Thanks about grayskull, I didn't know!